tests/Optics_comparison_images/generate_optics_comparison_images.py

```diff
@@ -87,7 +87,7 @@
     # 5 support struts in the pupil plane, of thickness 4% of the pupil diameter, with the
     # uppermost strut angled at 8 degrees to the vertical, with a touch of astigmatism, coma and
     # defocus
-    optics = galsim.OpticalPSF(
+    optics = galsim.optics.OpticalPSF(
```

Comment:
>A question that isn't about this code, but rather about the new files in tests/Optics_comparison_images/: I thought the new optics code reproduces the old one, so why have the files changed? Are there tiny numerical differences? Sorry if I'm missing something here.
 
Right.  @rmjarvis had a similar comment above.  At some point during development, I had decided these needed to be updated.  But I checked again, and the current code reproduces the original images on master now, so I reverted them back.  I'm not sure exactly what changed, but I think it's okay to just roll with it.  :)